docs/devs/Use-JerryScript-Debugger.md

```diff
@@ -12,6 +12,19 @@ intergrated into the binary of IoT.js.
 ### Usage
 
 To start the debugger-server: `<iotjs binary> --start-debug-server test.js`
+
+It is important to note that optional parameters (such as `--debugger-wait-source` or
+`--jerry-debugger-port=...`) should be called after `--start-debug-server` in order to work properly.
```

Comment:
called? this is argument list not function call :)
* should be specified after
* should come after